### PR TITLE
PolishFilter+Install+Instruction+En

### PR DESCRIPTION
**Pull request by Chinaski1**
Instruction Update

### CODE REVIEW

File: 05.android/02.installation/docs.en.md [hunk 1]
```diff
@@ -47,11 +47,41 @@ If you use Android 4, next you will see the list of permissions required by the
...
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Android3.1/installmainEn.png" />
+
+When you select the first option, a dialog box opens where you have to choose the level of privacy protection: 
+- **Comfortable** (Blocks all known Analytics , reduces the time of third-party cookies).
+- **High** (Reduces cookie lifetime, removes tracking parameters from URLS).
+- **Ultimate** (Disables browser features that can be used for your tracking. In some cases, you may experience problems with the operation of the sites).
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Android3.1/privacylevelEn.png" />
+
+**High** as well as **Ultimate** protection levels available only in **Premium** version.
```
Comment (vbagirov) [inline 05.android/02.installation/docs.en.md]: **High** as well as **Ultimate** protection levels are available only in **Premium** version.

File: 05.android/02.installation/docs.en.md [hunk 2]
```diff
@@ -47,11 +47,41 @@ If you use Android 4, next you will see the list of permissions required by the
...
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Android3.1/installpersonaldataEn.png" />
+
+In the next window, you will be given the option to hide irritant elements such as the **Like** and **Social media Widgets**.
```
Comment (vbagirov) [inline 05.android/02.installation/docs.en.md]: In the next window, you will be given the option to hide irritating elements such as the **Like** and **Social media Widgets**.

File: 05.android/02.installation/docs.en.md [hunk 3]
```diff
@@ -47,11 +47,41 @@ If you use Android 4, next you will see the list of permissions required by the
...
+When you select the first option, a dialog box opens where you have to choose the level of privacy protection: 
+- **Comfortable** (Blocks all known Analytics , reduces the time of third-party cookies).
+- **High** (Reduces cookie lifetime, removes tracking parameters from URLS).
+- **Ultimate** (Disables browser features that can be used for your tracking. In some cases, you may experience problems with the operation of the sites).
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Android3.1/privacylevelEn.png" />
+
+**High** as well as **Ultimate** protection levels available only in **Premium** version.
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Android3.1/installpersonaldataEn.png" />
```
Comment (vbagirov) [inline 05.android/02.installation/docs.en.md]: Про этот сркин ничего не написано.

File: 05.android/02.installation/docs.en.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ You will see the request to save the downloaded file. Press **OK**.
 
 When the download is complete, the system will ask you if you want to install the AdGuard app. Press **Install**.
 
-![](android_install_EN_04.png?cropResize=400,600)
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Android3.1/InstallMenuEn.png" />
 
 In Android 4, after AdGuard has been downloaded, you may see a dialogue window requesting for further action regarding the file. Select **Package installer** and then **Just once**.
```
Comment (vbagirov) [inline 05.android/02.installation/docs.en.md]: 4 версию мы не поддерживаем больше. Только 5+.

File: 02.general/01.adguard-ad-filters/docs.en.md [hunk 5]
```diff
@@ -178,7 +178,7 @@ Filter that enables removing of the ads from websites with Turkish content. Cre
 
 <a name="polish-ads"></a>
 
-## Polish ads filter
+## Polish Anti Adblock Filter
```
Comment (ameshkov) [inline 02.general/01.adguard-ad-filters/docs.en.md]: Guys, this is a third-party filter list, why is it here?

File: 05.android/02.installation/docs.en.md [hunk 6]
```diff
@@ -37,21 +37,46 @@ You will see the request to save the downloaded file. Press **OK**.
...
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Android3.1/installmainEn.png" />
 
-Wait for the installation to complete and press **Done**.
+When you select the first option, a dialog box opens where you have to choose the level of privacy protection.
 
-![](android_install_EN_07.png?cropResize=400,600)
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Android3.1/installpersonaldataEn.png" />
 
-AdGuard is now installed on your device. You can launch the app by pressing its icon in the list of installed apps.
+After pressing the **Block them** button, you will be offered three levels of protection:
```
Comment (vbagirov) [inline 05.android/02.installation/docs.en.md]: перенос строки перед перечислением уровней

File: 05.android/02.installation/docs.en.md [hunk 7]
```diff
@@ -37,21 +37,46 @@ You will see the request to save the downloaded file. Press **OK**.
...
+You will then be asked to read the **License agreement** and AdGuard's privacy policy. 
+You can participate in product development. 
+To do this, you should check the box *"Allow AdGuard to send automatic error reports"*.
+After that, you can pre-configure AdGuard protection by clicking **Continue**, or you can do it later by clicking **Skip quick settings**.
 
-![](android_install_EN_06.png?cropResize=400,600)
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Android3.1/installmainEn.png" />
 
-Wait for the installation to complete and press **Done**.
+When you select the first option, a dialog box opens where you have to choose the level of privacy protection.
```
Comment (vbagirov) [inline 05.android/02.installation/docs.en.md]: If you select the first option, a new dialog box will appear where you can configure privacy protection settings.

File: 05.android/02.installation/docs.en.md (hunk 6 above)
Comment (vbagirov) [reply 05.android/02.installation/docs.en.md]: И ещё можно изменить фразу слегка: 
After pressing the **Block them** button, you will be offered to choose between three levels of protection:

**Pull request merged**